.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -338,6 +338,11 @@ AddType text/vtt                            vtt
     # GitOps landing page
     RewriteRule ^container-platform/(4\.11|4\.12|4\.13|4\.14|4\.15|4\.16|4\.17|4\.18)/cicd/gitops/about-redhat-openshift-gitops.html /gitops/latest/understanding_openshift_gitops/about-redhat-openshift-gitops.html  [L,R=302]
 
+    # GitOps release notes version switch
+
+    RewriteRule ^gitops/1\.(8|9|10|11|12|13)/release_notes/gitops-release-notes?(.*).html gitops/1.$1/release_notes/gitops-release-notes.html
+    RewriteRule ^gitops/1\.(14|15|16)/release_notes/gitops-release-notes?(.*).html gitops/1.$1/release_notes/gitops-release-notes-1-$1.html
```

Comment:
These rules also catch existing links for example `https://docs.openshift.com/gitops/1.12/release_notes/gitops-release-notes.html` will match the first rule and point to the same link `https://docs.openshift.com/gitops/1.12/release_notes/gitops-release-notes.html`

```suggestion
    RewriteRule ^gitops/1\.(8|9|10|11|12|13)/release_notes/gitops-release-notes-1-([0-9]+)\.html$ gitops/1.$1/release_notes/gitops-release-notes.html [L, R302]
    RewriteRule ^gitops/1\.(14|15|16)/release_notes/gitops-release-notes\.html$ gitops/1.$1/release_notes/gitops-release-notes-1-$1.html [L, R302]
    
    **suggestion:** 💡We should use the complete URLs
```

Reply:
I haven't seen that flag syntax before. Do they need to be `[L,R=302]` with the equals sign?

Reply:
Upon careful thought the second change does not work, because switches can happen between versions with the new order too. So, for example, https://docs.openshift.com/gitops/1.14/release_notes/gitops-release-notes-1-15.html must change to https://docs.openshift.com/gitops/1.14/release_notes/gitops-release-notes-1-14.html

here is the line for OCP achieving this effect

`RewriteRule ^container-platform\/4\.(.+)\/release_notes\/ocp-4-((?!\1).+)-release-notes.html /container-platform/4.$1/release_notes/ocp-4-$1-release-notes.html [NE,R=301] (edited) `

PTAL now. I made the one change that does clearly work plus added Pipelines, which needs more lines because the path in the URL also changes.

If "redirect to same URL" is not an acceptable effect, I am not sure what can be done, apart from a separate redirect for every version. But for OCP, this one line somehow works. I can't really understand what it does - regexp are a mystery I am failing to crack